descartes_utilities/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package>
```

Comment:
Might be nice to start using [package format 2](http://www.ros.org/reps/rep-0140.html).